Cargo.toml

```diff
@@ -135,36 +79,12 @@ aws-sdk-dynamodb = { version = "1.63", optional = true }
-opentelemetry_0_24_pkg = { package = "opentelemetry", version = "0.24", optional = true }
-opentelemetry_0_25_pkg = { package = "opentelemetry", version = "0.25", optional = true }
-opentelemetry_0_26_pkg = { package = "opentelemetry", version = "0.26", optional = true }
-opentelemetry_0_27_pkg = { package = "opentelemetry", version = "0.27", optional = true }
-opentelemetry_0_28_pkg = { package = "opentelemetry", version = "0.28", optional = true }
-opentelemetry_0_29_pkg = { package = "opentelemetry", version = "0.29", optional = true }
 opentelemetry_0_30_pkg = { package = "opentelemetry", version = "0.30", optional = true }
-opentelemetry_sdk_0_21_pkg = { package = "opentelemetry_sdk", version = "0.21", optional = true }
-opentelemetry_sdk_0_22_pkg = { package = "opentelemetry_sdk", version = "0.22", optional = true }
-opentelemetry_sdk_0_23_pkg = { package = "opentelemetry_sdk", version = "0.23", optional = true }
-opentelemetry_sdk_0_24_pkg = { package = "opentelemetry_sdk", version = "0.24", optional = true }
-opentelemetry_sdk_0_25_pkg = { package = "opentelemetry_sdk", version = "0.25", optional = true }
-opentelemetry_sdk_0_26_pkg = { package = "opentelemetry_sdk", version = "0.26", optional = true }
-opentelemetry_sdk_0_27_pkg = { package = "opentelemetry_sdk", version = "0.27", optional = true }
-opentelemetry_sdk_0_28_pkg = { package = "opentelemetry_sdk", version = "0.28", optional = true }
-opentelemetry_sdk_0_29_pkg = { package = "opentelemetry_sdk", version = "0.29", optional = true }
+opentelemetry_0_31_pkg = { package = "opentelemetry", version = "0.31", optional = true }
 opentelemetry_sdk_0_30_pkg = { package = "opentelemetry_sdk", version = "0.30", optional = true }
-tracing-opentelemetry_0_22_pkg = { package = "tracing-opentelemetry", version = "0.22", optional = true }
-tracing-opentelemetry_0_23_pkg = { package = "tracing-opentelemetry", version = "0.23", optional = true }
-tracing-opentelemetry_0_24_pkg = { package = "tracing-opentelemetry", version = "0.24", optional = true }
-tracing-opentelemetry_0_25_pkg = { package = "tracing-opentelemetry", version = "0.25", optional = true }
-tracing-opentelemetry_0_26_pkg = { package = "tracing-opentelemetry", version = "0.26", optional = true }
-tracing-opentelemetry_0_27_pkg = { package = "tracing-opentelemetry", version = "0.27", optional = true }
-tracing-opentelemetry_0_28_pkg = { package = "tracing-opentelemetry", version = "0.28", optional = true }
-tracing-opentelemetry_0_29_pkg = { package = "tracing-opentelemetry", version = "0.29", optional = true }
-tracing-opentelemetry_0_30_pkg = { package = "tracing-opentelemetry", version = "0.30", optional = true }
+opentelemetry_sdk_0_31_pkg = { package = "opentelemetry_sdk", version = "0.31", optional = true }
 tracing-opentelemetry_0_31_pkg = { package = "tracing-opentelemetry", version = "0.31", optional = true }
+tracing-opentelemetry_0_32_pkg = { package = "tracing-opentelemetry", version = "0.32", optional = true }
```

Comment:
Sure we this ? I see that last one is `0.32.0` but from the Changelog it seems to me that we are updating to `0.31.0` ... but then, why don't we update to `0.32.0` if available ?

Reply:
https://github.com/open-telemetry/opentelemetry-rust/releases
https://github.com/tokio-rs/tracing-opentelemetry/releases